src/coreclr-debug/install.ts

```diff
@@ -71,6 +71,28 @@ export class DebugInstaller {
         let manifestString = fs.readFileSync(manifestPath, 'utf8');
         let manifestObject = JSON.parse(manifestString);
 
+        manifestObject.activationEvents = [
```

Comment:
Per Dustin's comment, this can wait for the next PR, but this needs to go to a different spot - I think we need to rewrite `activationEvents` at the very start of our activation handler if it is still set to `[ "*" ]` to make sure that we will not continuously try to download things in the case that we don't succeed in downloading the debugger.